mockall/tests/automock_foreign_c.rs

```diff
@@ -25,25 +30,27 @@ pub fn normal_usage() {
 #[should_panic(expected = "mock_ffi::foo1(5): No matching expectation found")]
 fn with_no_matches() {
     let ctx = mock_ffi::foo1_context();
-    ctx.expect()
-        .with(predicate::eq(4))
-        .returning(i64::from);
-    unsafe{ mock_ffi::foo1(5) };
+    ctx.expect().with(predicate::eq(4)).returning(i64::from);
```

Comment:
Please don't combine content changes and formatting changes in the same commit.  You can undo the change to this function.